index-data/data.js

```diff
@@ -1689,6 +1689,16 @@ grids_tables = {
 
 
 "requirements": [
+	{ 	"title":"Chinese Layout Requirements",
+		"sections": [ 
+			{"title":"Handling of Grid Alignment in Chinese and Western Mixed Text Composition", "url":"https://www.w3.org/TR/clreq/#handling_of_grid_alignment_in_chinese_and_western_mixed_text_composition"}
```

Comment:
seems pointing wrong section?

some exists in 2.4.2.3,
https://www.w3.org/TR/clreq/#id71

but see no subsection on table in section 4?
https://www.w3.org/TR/clreq/#positioning_of_headings_notes_illustrations_and_tables

Reply:
I added a link to 3.2.5 because this section (in the language enablement index) is about grids & tables.

I didn't add 2.4.2.3 because that section was mainly about differences between horizontal and vertical writing modes, not tables.

There is no subsection on tables in section 4 because we haven't written it yet (but we plan to write it).

We can split grids and tables into two separate sections if needed. After all, grid systems are not unique to CJK. They also exist in Western texts (see [Grid systems](https://www.amazon.com/Grid-systems-graphic-design-communication/dp/3721201450) and [欧文書体 2 定番書体と演出法](https://www.amazon.co.jp/%E6%AC%A7%E6%96%87%E6%9B%B8%E4%BD%93-2-%E5%AE%9A%E7%95%AA%E6%9B%B8%E4%BD%93%E3%81%A8%E6%BC%94%E5%87%BA%E6%B3%95-%E3%82%BF%E3%82%A4%E3%83%9D%E3%82%B0%E3%83%A9%E3%83%95%E3%82%A3%E3%81%AE%E5%9F%BA%E6%9C%ACBOOK-%E5%B0%8F%E6%9E%97/dp/4568503647)) and maybe some other scripts.